details/encryption.md

```diff
@@ -155,27 +158,35 @@ composed from:
 1. The fixed string "private-attribution", encoded in ASCII, terminated with a
    single zero-valued byte.
 
-2. The [ASCII serialization of the helper party
+2. The [ASCII serialization of the match key provider
    origin](https://datatracker.ietf.org/doc/html/rfc6454#section-6.2),
    terminated with a single zero-valued byte.
 
-3. The [ASCII serialization of the current site
+3. The [ASCII serialization of the helper party
    origin](https://datatracker.ietf.org/doc/html/rfc6454#section-6.2),
    terminated with a single zero-valued byte.
 
-4. The single-byte key identifier from the key configuration for the helper
+4. The [ASCII serialization of the registrable domain for the current
+   site](https://url.spec.whatwg.org/#host-registrable-domain) encoded in ASCII
+   as a period-separated sequence of
+   [A-labels](https://datatracker.ietf.org/doc/html/rfc5890#section-2.3.2.1),
+   terminated with a single zero-valued byte.
+
+5. The single-byte key identifier from the key configuration for the helper
    party.
 
-5. The current epoch, encoded as an two-byte integer in network byte order.
+6. The current epoch, encoded as an two-byte integer in network byte order.
 
 This produces the following process in pseudocode:
 
 ```python
-def ipa_info(helper_origin, site_origin, key_id, epoch):
+def ipa_info(mkp_origin, helper_origin, site_origin, key_id, epoch):
     return concat(encode_str("private-attribution"),
+              ascii_origin(mkp_origin),
```

Comment:
I vaguely remember us talking about putting null byte delimiter between domain and the next field